announcements/2020-Week-32/kvm.md

```diff
@@ -9,7 +9,7 @@ There are no KVM platform releases this week.
 
 **Web Interface Releases**
 
-1. [Happa v0.10.36](https://github.com/giantswarm/happa/releases/tag/v0.10.35) makes cluster upgrades from within happa safer. It makes it no longer possible to trigger an upgrade for non-ready or already upgrading clusters. Additionally, the cluster release selection will no longer contain any empty rows.
+1. [Happa v0.10.36](https://github.com/giantswarm/happa/releases/tag/v0.10.36) makes cluster upgrades from within happa safer. It makes it no longer possible to trigger an upgrade for non-ready or already upgrading clusters. Additionally, the cluster release selection will no longer contain any empty rows.
```

Comment:
@ubergesundheit i changed the links to go to v0.10.36 instead of v0.10.35 let me know if it's the wrong thing to do

Reply:
It was correct!